README.md

```diff
@@ -21,7 +21,6 @@ The table classes provided in `chimedb.data_index` are:
 * ArchiveFile
 * ArchiveFileCopy
 * ArchiveFileCopyRequest
-* ArchiveInst
```

Comment:
Does alpenhorn2 really completely drop the `ArchiveInst` table?  How is the instrument tracked instead?

This is a major change to how Finder gets used.

Remember that `chimestack_corr`, `chimeN2_corr`, `chime26m_corr` are distinguished by their instrument.